fix-file-permissions.sh

```
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -x #<- for debugging
+
+public_path=${1:-docker/files}
+private_path=${2:-docker/private}
+www_id=${3:-82} # (Generally 33:33 for Debian and 82:82 for Alpine, which wodby uses)
+
+# PUBLIC: World readable, owned by PHP process (which for Wodby is www-data)
+find "${public_path}/" -type f ! -perm ug=rw,o=r -print0 | xargs -r -0 -L20 chmod ug=rw,o=r
+find "${public_path}/" -type d ! -perm ug=rwx,o=rx -print0 | xargs -r -0 -L20 chmod ug=rwx,o=rx
+find "${public_path}/" ! -user ${www_id} -print0 | xargs -r -0 -L20 chown ${www_id}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)